application/controllers/Lotw.php

```diff
@@ -723,6 +723,88 @@ function lotw_download($sync_user_id = null) {
 		}
 	}
 
+	public function check_lotw_credentials () {
+		$this->load->model('user_model');
+		if(!$this->user_model->authorize(2)) {
+			$this->session->set_flashdata('error', __("You're not allowed to do that!"));
+			redirect('dashboard');
+			exit();
+		}
+		$ret=[];
+		$ret['status']='';
+
+
+		$raw = file_get_contents("php://input");
+		try {
+			$obj = json_decode($raw,true);
+		} catch (e) {
```

Comment:
This will probably not work as expected as `e` is not defined. Should be defined as Exception event


```suggestion
		} catch (Exception $e) {
```

Reply:
Done too much JS the last time. Tnx